src/RemoteResourceController.js

```diff
@@ -22,7 +22,7 @@ const { BaseDownloadController } = require('@razee/razeedeploy-core');
 
 module.exports = class RemoteResourceController extends BaseDownloadController {
   constructor(params) {
-    params.finalizerString = params.finalizerString || 'children.remoteresource.deploy.razee.io';
+    params.finalizerString = params.finalizerString || 'children.downloads.deploy.razee.io';
```

Comment:
How will existing remote resources behave when we update the operator?

Reply:
they will break..

Reply:
i'll have to see about what i want to do about that..

Reply:
updated this and rrs3 controller file to both be `remoteresource` instead of `downloads`